two-phase-commit/src/Makefile

```diff
@@ -0,0 +1,35 @@
+BIN := bin/
+
+MSTR = $(BIN)tpc_master
+SLV = $(BIN)tpc_slave
+
+CFLAGS = -g -Wall
+
+MASTER_OBJ =  $(BIN)tpc_clnt.o $(BIN)tpc_xdr.o $(BIN)tpc_master.o
+SLAVE_OBJ = $(BIN)tpc_slave.o $(BIN)tpc_xdr.o $(BIN)tpc_svc.o
+
+all:
+	@rpcgen tpc.x
+	@make objects
+
+objects: $(SLV) $(MSTR)
+
+$(MSTR): $(MASTER_OBJ) tpc.h vote.h
+	@echo "compiling master"
+	gcc -o $(MSTR) $(MASTER_OBJ) -lpthread
```

Comment:
Is there a specific reason this (and the very next) invocation of `gcc` is not hidden with `@` like the rest?

Reply:
fixed